packages/react-native/ReactCommon/react/nativemodule/core/platform/ios/ReactCommon/RCTTurboModule.h

```diff
@@ -141,14 +147,15 @@ class JSI_EXPORT ObjCTurboModule : public TurboModule {
       bool isSync,
       const char *methodName,
       NSInvocation *inv,
-      NSMutableArray *retainedObjectsForInvocation);
+      NSMutableArray *retainedObjectsForInvocation,
+      RCTNSDictionaryPromiseRejectBlock optionalInternalRejectBlock);
```

Comment:
Could this be named more generically as RCTInvocationErrorHandler? The fact that it rejects a promise is specific to this invocation.